tests/test_more_ops.py

```diff
@@ -555,6 +555,29 @@ def input_shapes(self):
         return [(4, 4)]
 
 
+class ToCopyModule(torch.nn.Module):
```

Comment:
We really have to start splitting things into more logical files. 
"more_ops" is.. not good

Reply:
Yeah will definitely prioritize this during refactoring soon.